src/components/lib/mock-data.json

```diff
@@ -0,0 +1,26 @@
+[
+	{
+    "id": 1,
+		"distance": 20,
+		"time": "40:00",
+		"pace": "04:49",
+		"borg": 8,
+		"pain": 2
```

Comment:
Please check the indention.

Reply:
It's correctly indented for me. 🤔

<a href="https://ibb.co/kSs1SVL"><img src="https://i.ibb.co/fxzvxV3/Screenshot-2022-08-18-at-15-26-10.png" alt="Screenshot-2022-08-18-at-15-26-10" border="0"></a>